### PR TITLE
Hestia v1.0.2 beta

### PR DESCRIPTION
**Pull request by ioannidesalex**

### CODE REVIEW

File: install/upgrade/versions/1.0.2.sh [hunk 1]
```diff
@@ -6,4 +6,63 @@
...
+if [ ! -z "$WEBALIZER_CHECK" ]; then
+    echo "(*) Removing Webalizer and setting AWStats as default web statistics backend..."
+    apt purge webalizer -y > /dev/null 2>&1
+    if [ -d "$HESTIA/data/templates/web/webalizer" ]; then
+        rm -rf $HESTIA/data/templates/web/webalizer
+    fi
+    if [ -d "/var/www/webalizer" ]; then
+        rm -rf /var/www/webalizer
+    fi
+    sed -i "s/STATS_SYSTEM='webalizer,awstats'/STATS_SYSTEM='awstats'/g" $HESTIA/conf/hestia.conf
```
Comment (Lupul) [inline install/upgrade/versions/1.0.2.sh:48]: is the stats_system order always the same?

File: install/upgrade/versions/1.0.2.sh [hunk 2]
```diff
@@ -6,4 +6,63 @@
...
+fi
+
+# Remove old hestia.conf files from Apache & NGINX if they exist
+if [ -f "/etc/nginx/conf.d/hestia.conf" ]; then
```
Comment (Lupul) [inline install/upgrade/versions/1.0.2.sh]: typo?

File: install/upgrade/versions/1.0.2.sh [hunk 3]
```diff
@@ -6,4 +6,63 @@
...
+fi
+if [ -f "/etc/nginx/conf.d/hestia.conf" ]; then
+    echo "(*) Removing old NGINX configuration file from previous version of Hestia Control Panel..."
+    rm -f /etc/nginx/conf.d/hestia.conf
+fi
+
+# Update webmail templates to enable OCSP/SSL stapling
+if [ ! -z "$IMAP_SYSTEM" ]; then
+    echo "(*) Enabling OCSP stapling support for webmail services..."
+    $BIN/v-update-mail-templates > /dev/null 2>&1
```
Comment (Lupul) [inline install/upgrade/versions/1.0.2.sh:64]: We should backup old templates

**Review by Lupul — COMMENTED** [review]
Besides my comments, looks good
Don't know about the ssl parameter changes though (especially dovecot), how they are impacting compatibility with different devices (outlook express, android mail clients etc)

File: install/upgrade/versions/1.0.2.sh (hunk 1 above)
Comment (ioannidesalex) [reply install/upgrade/versions/1.0.2.sh:48]: Yes.

**Review by ghost — APPROVED** [review]

**Review by ScIT-Raphael — APPROVED** [review]

**Pull request merged**